src/test/java/com/google/devtools/build/lib/rules/platform/PlatformTest.java

```diff
@@ -35,8 +35,11 @@
   @Rule public ExpectedException expectedException = ExpectedException.none();
 
   @Test
+  // TODO(https://github.com/bazelbuild/bazel/issues/6849): Remove this test when the functionality
```

Comment:
Is this line still within the char limit? Unsure whether this can be visualized in github review...

Reply:
Yes, 99 characters long according to vim.